stacker/providers/aws/default.py

```diff
@@ -106,105 +273,71 @@ def cloudformation(self):
 
         return self._cloudformation
 
-    def get_stack(self, stack_name, **kwargs):
-        try:
-            return retry_on_throttling(
-                self.cloudformation.describe_stacks,
-                kwargs=dict(StackName=stack_name))['Stacks'][0]
-        except botocore.exceptions.ClientError as e:
-            if "does not exist" not in e.message:
-                raise
-            raise exceptions.StackDoesNotExist(stack_name)
+    @property
+    def listener(self):
+        # deals w/ multiprocessing issues w/ sharing ssl conns
+        # see https://github.com/remind101/stacker/issues/196
+        pid = os.getpid()
+        if pid != self._pid or not self._listener:
+            session = get_session(self.region)
+
+            # This stays the same for stacker call in this namespace
+            topic_name = self.namespace
```

Comment:
maybe prepend `stacker-`  just in case someone else tries to create an sns topic w/ the namespace.  Maybe not worth worrying about, but just the namespace may make people unsure of what this topic is.